dataactbroker/handlers/userHandler.py

```diff
@@ -177,83 +164,65 @@ def getUserPermissions(self, user):
         """ Checks if user has specified permission
 
         Arguments:
             user - User object
-            permissionName - permission to check
+            permission_name - permission to check
         Returns:
             True if user has the specified permission, False otherwise
         """
+        sess = GlobalDB.db().session
         # Get the bit number corresponding to this permission from the permission_types table
-        bitNumber = self.getPermissionId(permissionName)
+        bit_number = sess.query(PermissionType).filter(PermissionType.name == permission_name).one().permission_type_id
         # Use that bit number to check whether user has the specified permission
-        if checkPermissionByBitNumber(user, bitNumber):
+        if checkPermissionByBitNumber(user, bit_number):
             return True
         return False
 
-    def setPermission(self,user,permission):
-        """ Define a user's permission to set value (overwrites all current permissions)
-
-        Arguments:
-            user - User object
-            permission - new value for user's permissions
-        """
-        user.permissions = permission
-        self.session.commit()
-
-    def grantPermission(self,user,permissionName):
+    def grantPermission(self,user,permission_name):
         """ Grant a user a permission specified by name, does not affect other permissions
 
         Arguments:
             user - User object
-            permissionName - permission to grant
+            permission_name - permission to grant
         """
-        if(user.permissions == None):
+        sess = GlobalDB.db().session
+        if user.permissions is None:
             # Start users with zero permissions
             user.permissions = 0
-        bitNumber = self.getPermissionId(permissionName)
-        if not checkPermissionByBitNumber(user, bitNumber):
+        bit_number = sess.query(PermissionType).filter(PermissionType.name == permission_name).one().permission_type_id
+        if not checkPermissionByBitNumber(user, bit_number):
             # User does not have permission, grant it
-            user.permissions = user.permissions + (2 ** bitNumber)
+            user.permissions += (2 ** bit_number)
             self.session.commit()
 
-    def removePermission(self,user,permissionName):
+    def removePermission(self,user,permission_name):
         """ Remove a permission specified by name from user
 
         Arguments:
             user - User object
             permissionName - permission to remove
         """
-        if(user.permissions == None):
+        sess = GlobalDB.db().session
+        if user.permissions is None:
             # Start users with zero permissions
             user.permissions = 0
-        bitNumber = self.getPermissionId(permissionName)
-        if checkPermissionByBitNumber(user, bitNumber):
+        bit_number = sess.query(PermissionType).filter(PermissionType.name == permission_name).one().permission_type_id
```

Comment:
I feel like there are more comments than instances of this... but I think I got them all? You'll have to tell me later.